src/components/resumption/test/last_state_test.cc

```diff
@@ -64,55 +78,77 @@ class LastStateTest : public ::testing::Test {
   const std::string empty_dictionary_;
   const std::string app_info_dat_file_;
 
-  resumption::LastState last_state_;
+  ::utils::SharedPtr<resumption::LastState> last_state_;
+
+  void CompactJson(std::string& str) {
+    str.erase(remove_if(str.begin(), str.end(), isspace), str.end());
```

Comment:
@AByzhynar . Yes. In previuos realization also all spaces, `\n`, `\t' were deleteing.